### PR TITLE
disable inactive cards, click to focus

### PR DESCRIPTION
**Pull request by alansutherland**

### CODE REVIEW

File: components/item/AwesomeSwiper.vue [hunk 1]
```diff
@@ -59,6 +60,7 @@
           slidesPerView: 'auto',
           spaceBetween: 40,
           centeredSlides: true,
+          slideToClickedSlide: true,
```
Comment (mirjamverloop) [inline components/item/AwesomeSwiper.vue:63]: Is this a swiper option? Or is it used?

File: components/item/AwesomeSwiper.vue [hunk 2]
```diff
@@ -74,30 +76,43 @@
...
+      position: absolute;
+    }
     a {
       display: inline-flex;
       height: 100%;
       align-items: center;
     }
+    &:last-of-type {
+      margin-right: 50vw !important;
+    }
```
Comment (mirjamverloop) [inline components/item/AwesomeSwiper.vue:111]: As discussed, could you remove this for now?

Comment (alansutherland) [reply components/item/AwesomeSwiper.vue:111]: ok

File: components/item/AwesomeSwiper.vue (hunk 1 above)
Comment (alansutherland) [reply components/item/AwesomeSwiper.vue:63]: yes it's a built-in option which achieves this requirement: “When a resource is not the current focus image, clicking it brings it to the focus”

Comment (mirjamverloop) [reply components/item/AwesomeSwiper.vue:63]: Thanks for clarifying!

**Pull request merged**